### ISSUE TITLE
"Next file/episode" button in player that also marks as played?

### ISSUE THREAD

**Issue by Pentaphon** [nova-video-player/aos-AVP#443]
I'm not sure if you're planning a player UI overhaul which may include this feature, but I think it is important to add ASAP.

The age of streaming TV is moving faster than ever before and we're going to have more and more TV series than ever. Streaming services have an easy way for you to skip to the next episode, even if streaming player remotes don't have a "next track" button. 

Therefore, it is important that Nova include a way to manually move to the next file/episode in a folder/series without waiting for the file to end and also marks that episode/video as watched. 

I'm already aware of folder mode, but you have to wait for the file to end and sometimes, we want to skip manually so we don't have to view the end credits, which are even longer than ever due to subtitle credits also being added. 

Not sure what the best way to implement this considering the player UI is so simple. Something like this? Or anything else you can figure out that does the same function. Maybe when you press up on the player UI the first function on the menu is a "next episode + mark as played button"? 

![next](https://user-images.githubusercontent.com/49374058/115109781-a45f0480-9f2c-11eb-8b8c-05d12bd190d0.jpg)

**Comment by jungleclaptrap** [comment]
I still think this feature is needed, because currently you have to press 4 buttons (back button-> back button-> the next track -> watch from beginning) just to go to the next track, compare to just having 1 button to go to the next track like most video players.

**Comment by Pentaphon** [comment]
> I still think this feature is needed, because currently you have to press 4 buttons (back button-> back button-> the next track -> watch from beginning) just to go to the next track, compare to just having 1 button to go to the next track like most video players.

Yep! This might be a part of the "new v6 look" redesign but I'm not sure. We'd have to ask @courville if this feature is part of that redesign.
